src/main/java/org/opentripplanner/graph_builder/impl/shapefile/NihShapefileStreetGraphBuilderImpl.java

```diff
@@ -0,0 +1,457 @@
+
+        FeatureSource<SimpleFeatureType, SimpleFeature> featureSource = _featureSourceFactory
+                .getFeatureSource();
+
+        FeatureCollection<SimpleFeatureType, SimpleFeature> features = null;
+
+        try {
+            CoordinateReferenceSystem sourceCRS = featureSource.getInfo().getCRS();
+
+            Hints hints = new Hints(Hints.FORCE_LONGITUDE_FIRST_AXIS_ORDER, Boolean.TRUE);
+            CRSAuthorityFactory factory = ReferencingFactoryFinder.getCRSAuthorityFactory("EPSG",
+                    hints);
+            CoordinateReferenceSystem worldCRS = factory
+                    .createCoordinateReferenceSystem("EPSG:4326");
+
+            Query query = new Query();
+            query.setCoordinateSystem(sourceCRS);
+            query.setCoordinateSystemReproject(worldCRS);
+
+            features = featureSource.getFeatures(query);
+
+        } catch (Exception ex) {
+            throw new IllegalStateException("error loading shapefile street data", ex);
+        } finally {
+            _featureSourceFactory.cleanup();
+        }
+
+        // TODO: get rest of properties
+        StreetVertexIndexServiceImpl vertexSvc = new StreetVertexIndexServiceImpl(graph);
+        SimpleFeatureConverter<Long> osmIdSelector = new AttributeFeatureConverter("osm_id");
+        SimpleFeatureConverter<Integer> segmentIdSelector = new AttributeFeatureConverter("segment_id");
+        StringAttributeFeatureConverter nameSelector = new StringAttributeFeatureConverter("name", "");
+
+        List<SimpleFeature> featureList = new ArrayList<SimpleFeature>();
+        FeatureIterator<SimpleFeature> it2 = features.features();
+        while (it2.hasNext()) {
+            SimpleFeature feature = it2.next();
+            featureList.add(feature);
+        }
+        it2.close();
+        it2 = null;
+
+        LOG.info("Have {} NIH features to process...", featureList.size());
+
+        // index edges and nodes in graph built from OSM data by their OSM IDs
+        LOG.info("Indexing OSM nodes and ways by ID...");
+        Multimap<String, StreetEdge> edgesByWayId = HashMultimap.create();
+        Map<String, IntersectionVertex> intersectionsByNodeId = Maps.newHashMap();
+        Set<Vertex> vertices = Sets.newHashSet();
+        Collection<Edge> edges = graph.getEdges();
+        for (Edge edge: edges) {
+            if (edge instanceof StreetEdge) {
+                StreetEdge street = (StreetEdge) edge;
+                edgesByWayId.put(street.getOsmId(), street);
+                vertices.add(street.getFromVertex());
+                vertices.add(street.getToVertex());
+            }
+        }
+
+        edges = null;
+
+        for (Vertex vertex : vertices) {
+            if (vertex instanceof IntersectionVertex) {
+                IntersectionVertex intersection = (IntersectionVertex) vertex;
+                intersectionsByNodeId.put(intersection.getLabel(), intersection);
```

Comment:
It doesn't look like this actually gets used anywhere, maybe it's a leftover from something earlier?